classes/main.rb

```diff
@@ -1,11 +1,12 @@
 require_relative 'app'
-
+require_relative 'execute_option'
 app = App.new
 
+# rubocop:disable Metrics/CyclomaticComplexity
 def main(app)
```

Comment:
- [ ] As required by Microverse, you are not meant to use any linters disable rule in your code. The linters are there to make sure your code follows the best practices, I'll suggest you remove this and refactor your code to fix the linter error. :small_airplane: :wave: :smile:

Reply:
@V-Blaze Microverse allow linter disabling rules for errors they just don't allow complete manipulation

Reply:
@V-Blaze reference https://github.com/microverseinc/linters-config